modules/oadp-timeouts.adoc

```diff
@@ -0,0 +1,194 @@
+
+.Procedure
+* Edit the values in the `spec.configuration.velero.resourceTimeout` block of the `DataProtectionApplication` CR manifest, as in the following example:
+[source,yaml]
+----
+apiVersion: oadp.openshift.io/v1alpha1
+kind: DataProtectionApplication
+metadata:
+ name: <dpa_name>
+spec:
+  configuration:
+    velero:
+      resourceTimeout: 10m
+  ...
+----
+
+[id="Datamover-timeout_{context}"]
+== Data Mover timeout
+
+.Description
+`timeout` is a user supplied timeout to be used for `VolumeSnapshotBackup` and `VolumeSnapshotRestore` to complete. The default value is 10m.
+
+.Implementation
+.. Use this timeout when VSB/VSR creation times out after 10min, use this setting to adjust the timeout appropriately.
+.. For large scale environments set timeout for 1hr.
+.. Use this timeout with the vsm plugin.
+.. Use this timeout with OADP 1.1.x.
+
+.Procedure
+* Edit the values in the `spec.features.dataMover.timeout` block of the `DataProtectionApplication` CR manifest, as in the following example:
+[source,yaml]
+----
+apiVersion: oadp.openshift.io/v1alpha1
+kind: DataProtectionApplication
+metadata:
+ name: <dpa_name>
+spec:
+  features:
+    dataMover:
+      timeout: 10m
+  ...
+----
+
+[id="CSIsnapshot-timeout_{context}"]
+== CSI snapshot timeout
+
+.Description
+`CSISnapshotTimeout` specifies the time used to wait for `CSI VolumeSnapshot`, the status turns to `ReadyToUse` during creation, before returning error as timeout. The default value is 10m.
+
+.Implementation
+.. Use this timeout with the csi plugin.
+.. Very large volumes may take longer than 10 minutes to snapshot, adjust this timeout if timeouts are found in the logs.
+.. It is unlikely that this timeout will require adjusting.
+
+.Procedure
+* Edit the values in the `spec.csiSnapshotTimeout` block of the `Backup` CR manifest, as in the following example:
+[source,yaml]
+----
+apiVersion: velero.io/v1
+kind: Backup
+metadata:
+ name: <backup_name>
+spec:
+ csiSnapshotTimeout: 10m
+  ...
+----
+
+[id="Velero-default-item-operation-timeout_{context}"]
+== Velereo default item operation timeout
+
+.Description
+`defaultItemOperationTimeout` defines how long to wait on asynchronous `BackupItemActions` and `RestoreItemActions` to complete before timing out. The default value is 1h.
+
+.Implementation
+.. Use this timeout only with Data Mover 1.2.x.
+.. Use this timeout to specify the amount of time a particular backup or restore should wait for the Asynchronous actions to complete. In the context of OADP features, this value is used for the Asynchronous actions involved in the Container Storage Interface (CSI) Data Mover feature.
+.. When `defaultItemOperationTimeout` is defined in the DPA using the `defaultItemOperationTimeout`, it applies to both backup and restore operations. You can use `itemOperationTimeout` to define only the backup or only the restore of those CRs, as described below in *Item operation timeout - restore*, and *Item operation timeout - backup*.
```

Comment:
```suggestion
.. When `defaultItemOperationTimeout` is defined in the DPA by using the `defaultItemOperationTimeout`, [vague referrent: three nouns stated before this dependent clause] applies to both backup and restore operations. You can use `itemOperationTimeout` to define only the backup or only the restore of those CRs, as described below in *Item operation timeout - restore*, and *Item operation timeout - backup*.
```

Reply:
Avoid positional words such as "below"; best to use the standard clause "the following".